library/kotlin/src/io/envoyproxy/envoymobile/EnvoyStreamEmitter.kt

```diff
@@ -50,6 +50,7 @@ class EnvoyStreamEmitter(
    * @throws EnvoyException when there is an exception cancelling the stream.
    */
   override fun cancel() {
+    // TODO: surface result of operation.
```

Comment:
tbh I don't know that we need to, since we don't/can't surface the synchronous results of any other of our other stream operations.

Reply:
(with the current interfaces, anyways)